org.elysium.test/src/org/elysium/test/regression/TrillTest.xtend

```diff
@@ -0,0 +1,34 @@
+package org.elysium.test.regression
+
+import org.junit.runner.RunWith
+import org.eclipse.xtext.junit4.XtextRunner
+import org.eclipse.xtext.junit4.InjectWith
+import org.elysium.LilyPondInjectorProvider
```

Comment:
I have an Xtend error here: org.elysium.LilyPondInjectorProvider cannot be resolved to a type.